src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,10 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private static final UserService userService = new UserService();
```

Comment:
I was thinking that it would be better to not create a new object of UserService whenever we create an object of AuthenticationService.

Reply:
When we work with objects, most of the time we want to avoid static because it is hard to test and it is a bit contradictory if we think about the OOP, which we will cover later on in the course :)